swiperefresh/src/main/java/com/google/accompanist/swiperefresh/IndicatorSize.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.accompanist.swiperefresh
+
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+/**
+ * A enumerated class to encapsulate details of different indicator sizes.
+ *
+ * @param size The overall size of the indicator.
+ * @param arcRadius The radius of the arc.
+ * @param strokeWidth The width of the arc stroke.
+ * @param arrowWidth The width of the arrow.
+ * @param arrowHeight The height of the arrow.
+ */
+enum class IndicatorSize(
```

Comment:
I think this could be a normal data class instead, with `Default` and `Large` being constants in the companion object. This allows devs to provide their own values if they wish.